addon/utils/clamp.js

```diff
@@ -10,18 +10,33 @@
  * @param {String} cssClass - A CSS class applied to the last line instead of inline CSS.
  */
 
-var measure, text, lineWidth, pos,
-  lineStart, lineCount, wordStart,
-  line, lineText, wasNewLine,
-  nodeStack, seedQueue, pendingQueue,
-  textNode, measureWidth, thisNode, nextQueue,
-  ce, ctn;
+var measure,
+  text,
+  lineWidth,
+  pos,
+  lineStart,
+  lineCount,
+  wordStart,
+  line,
+  lineText,
+  wasNewLine,
+  nodeStack,
+  seedQueue,
+  pendingQueue,
+  textNode,
+  measureWidth,
+  thisNode,
+  nextQueue,
+  ce,
+  ctn;
```

Comment:
Can we make these not be single var declarations? Also, `var` 🤯. Let's move to `let`.

Reply:
This change was automated by prettier.

Reply:
I'm actually curious why there isn't a linting rule for avoiding `var`...

Reply:
Ya I know it was made by prettier, but there's a var in there. I'm pretty sure there's a rule that forbids this (and may have an auto-fixer!).

Reply:
Yea I fixed the `var` already :)

Reply:
ty